sys-libs/libomp/libomp-9999.ebuild

```diff
@@ -17,16 +18,26 @@ EGIT_REPO_URI="http://llvm.org/git/openmp.git
 LICENSE="UoI-NCSA"
 SLOT="0"
 KEYWORDS=""
-IUSE=""
+IUSE="test"
 
-DEPEND="dev-lang/perl"
+DEPEND="dev-lang/perl
+	test? ( ${PYTHON_DEPS} )"
+
+pkg_setup() {
+	use test && python-any-r1_pkg_setup
+}
 
 multilib_src_configure() {
 	local libdir="$(get_libdir)"
 	local mycmakeargs=(
 		-DLIBOMP_LIBDIR_SUFFIX="${libdir#lib}"
```

Comment:
Why not `LLVM_LIBDIR_SUFFIX`? I recall you intend to use this consistently throughout all llvm components?